PROJECT

```diff
@@ -3,9 +3,15 @@ layout: go.kubebuilder.io/v2
 projectName: observability-operator
 repo: github.com/redhat-developer/observability-operator
 resources:
-- group: observability
+-
+  # TODO(user): Uncomment the below line if this resource implements a controller, else delete it.
+  # controller: true
+  domain: redhat.com
+  group: observability
   kind: Observability
+  # TODO(user): Update the package path for your API if the below value is incorrect.
+  path: github.com/redhat-developer/observability-operator/v3/api/v1
```

Comment:
Is this PR is till "WIP" and should these TODOs be acted on? :thinking:

Reply:
Thank you very much for comment. Subject updated now (forgot : ). Regarding `PROJECT` file - it was generated, as changes required moving from version `3-alpha` to `3`. I will try recheck it with OO people before removing `TODO` comments. Thanks again for comment @makslion

Reply:
@makslion Thank you for comments. Update done. OO bundles created now with operator-sdk 1.4.2 (as defined in OO in observability-operator.clusterserviceversion.yaml, annotations.yaml). PROJECT file (and few  others) restored as in main branch